src/common/runtime/src/lib.rs

```diff
@@ -17,6 +17,8 @@ pub mod global;
 mod metrics;
 mod repeated_task;
 pub mod runtime;
+mod runtime_default;
+mod runtime_throttle_count_mode;
```

Comment:
For this throttleable capability, the default spawn runtime has the highest priority, meaning there will be no scheduling restrictions on tasks; thus, it will not affect the original capabilities. Additionally, implementing this as a feature will lead to version divergence. For a large project like Greptime, switching features may cause the logic of basic library calls to diverge, which means conditional judgments for features would need to be introduced in the core code. Personally, I think this is unnecessary.
```suggestion
mod runtime_default;
mod runtime_throttleable;
```

Reply:
For this throttleable capability, the default spawn runtime has the highest priority, meaning there will be no scheduling restrictions on tasks; thus, it will not affect the original capabilities. Additionally, implementing this as a feature will lead to version divergence. For a large project like Greptime, switching features may cause the logic of basic library calls to diverge, which means conditional judgments for features would need to be introduced in the core code. Personally, I think this is unnecessary.